blocks/tabs/tabs.js

```diff
@@ -27,6 +46,30 @@ export default async function decorate(block) {
       tabpanel.lastElementChild.innerHTML = `<p>${tabpanel.lastElementChild.innerHTML}</p>`;
     }
 
+    // decorate faq item label
+    const label = tabpanelCopy.children[1].querySelector('h3');
+    const summary = document.createElement('summary');
```

Comment:
I am not sure if adding custom tags are correct or not from SEO perspective.
Why are we not handling with the classes here